pkg/harvester/components/FilterVMSchedule.vue

```diff
@@ -25,6 +25,9 @@ export default {
 
       return Array.from(new Set(options));
     },
+    enableFilterButton() {
+      return this.rows.some((r) => r.sourceSchedule !== undefined);
```

Comment:
I think we are fixing this in another PR https://github.com/harvester/dashboard/pull/1263